pkg/pool-stable-phantom/test/StablePhantomPool.test.ts

```diff
@@ -810,7 +846,7 @@ describe('StablePhantomPool', () => {
                 recipient: lp,
               });
 
-              expect(amountIn).to.be.equalWithError(queryResult.amountsIn[tokenIndex], 0.00001);
+              expect(amountIn).to.be.equal(queryResult.amountsIn[tokenIndex]);
```

Comment:
Were these failing prior to the change?

Reply:
Yes; if you switch them to "equal" in current master, they fail.